reana_commons/utils.py

```diff
@@ -15,6 +15,7 @@
 import subprocess
 import time
 import uuid
+import fs
```

Comment:
Now that we will use `fs` from R-Commons, it is a good moment to centralise the dependency, can you add it to `setup.py` (otherwise `reana-client` crashes on `run-ci`) and remove it from [RS](https://github.com/reanahub/reana-server/blob/514720b06e3c147d685ee127303510da6bb458a5/setup.py#L49) and [RWC](https://github.com/reanahub/reana-workflow-controller/blob/cf18f6f75920a28ad0744cea692115ddc51b92dc/setup.py#L50)?